include/swift/IDE/CodeCompletion.h

```diff
@@ -533,6 +533,9 @@ enum class CompletionKind {
   None,
   Import,
   UnresolvedMember,
+  /// An unresolved member completion in a pattern matching position.
+  /// E.g. \code if case .#^COMPLETE^# = x {} \endcode
+  PatternMatching,
```

Comment:
I'm not a fan of this naming. Could you include something that imply "after dot" and "without base expr/pattern"? For example `PatternAfterDotWithoutBase` or `UnresolvedDotPattern`?

Reply:
Renamed to `UnresolvedDotPattern`.